Gems/LuaScript/Code/Source/Previewer/ui_LuaScriptPreviewer.h

```diff
@@ -0,0 +1,86 @@
+/********************************************************************************
+** Form generated from reading UI file 'LuaScriptPreviewer.ui'
+**
+** Created by: Qt User Interface Compiler version 6.1.2
+**
+** WARNING! All changes made in this file will be lost when recompiling UI file!
+********************************************************************************/
```

Comment:
This file is generated by the Qt UIC tool through cmake, it does not need to be committed to source control